eZ/Publish/Core/MVC/Symfony/Matcher/ContentBased/Id/ParentContentType.php

```diff
@@ -56,9 +56,13 @@ public function match(View $view)
         if (!$view instanceof LocationValueView) {
             return false;
         }
-        $parent = $this->loadParentLocation(
-            $view->getLocation()->parentLocationId
-        );
+        try {
+            $parent = $this->loadParentLocation(
+                $view->getLocation()->parentLocationId
+            );
+        } catch (\eZ\Publish\API\Repository\Exceptions\NotFoundException $e) {
```

Comment:
rather `\Throwable` as well. NotFound should not occur here, unless someone explicitly used non-existent location id.